javascript/UserForm.js

```diff
@@ -63,6 +63,13 @@ jQuery(function ($) {
 
 		this.$el.validate(this.validationOptions);
 
+		// Ensure checkbox groups are validated correctly
```

Comment:
Does this make each option in the checkbox set required? You should only need to select at least one option if it's a required field (otherwise, it's not an optionset is it, it's a requiredset. :) )